models/project/project.go

```diff
@@ -115,6 +199,25 @@ func (u *ProjectStruct) Lists() []*list.ListStruct {
 	return slice
 }
 
+func (u *ProjectStruct) NoneList() *list.ListStruct {
+	table := u.database.Init()
+	defer table.Close()
+
+	var id, projectId, userId int64
+	var title, color sql.NullString
+	var optionId sql.NullInt64
+	err := table.QueryRow("select id, project_id, user_id, title, color, list_option_id from lists where project_id = ? and title = ?;", u.Id, config.Element("init_list").(map[interface{}]interface{})["none"].(string)).Scan(&id, &projectId, &userId, &title, &color, &optionId)
+	if err != nil {
+		// noneが存在しないということはProjectsController#Createがうまく行ってないので，そっちでエラーハンドリングしてほしい
+		panic(err)
+	}
+	if projectId == u.Id && title.Valid {
+		return list.NewList(id, projectId, userId, title.String, color.String, optionId)
+	} else {
```

Comment:
__[gometalinter]__
* [golint] if block ends with a return statement, so drop this else and outdent its block